app/classifier/tasks/text/index.jsx

```diff
@@ -73,7 +74,7 @@ export default class TextTask extends React.Component {
 
   handleChange() {
     const value = this.textInput.current.value;
-    if (value < this.state.value) {
+    if (value.length < this.state.value.length) {
       this.setState({ rows: 1, value }, () => { this.handleResize(); });
```

Comment:
The callback can just be `this.handleResize` here. It doesn’t need to be wrapped in an anonymous function.